### DISCUSSION TITLE
Can the leaderboard code be obtimized using sockets ?

### DISCUSSION THREAD

**Discussion by heysm** [sa-y-an/FinalArcane#5] [Q&A] [answered]
- Each time some one checks the leader board a query is made 
- Can't this be optimized using sockets

**Comment by sa-y-an** [comment, marked as answer]
> * Each time some one checks the leader board a query is made
> * Can't this be optimized using sockets

- Actually, since the number of participants is significantly less
- The number of refreshes will be small, thus I didn't complicate with sockets 
- Instead, I am planning to use Redis cache the leaderboard, and query every 5 mins or so
